quickwit/quickwit-config/src/source_config.rs

```diff
@@ -158,6 +170,22 @@ impl SourceConfig {
             _ => None,
         }
     }
+
+    pub fn enabled(&self) -> bool {
```

Comment:
I Have removed it for now. But I think we'd better hide all of them because things like num_pipelines have special treatment for certain source types and they are still pub so are best to be accessed via accessor methods.